src/cover/classify.py

```diff
@@ -31,14 +31,14 @@ def lookup_classification(name_lookup, crop_scape_value):
 
 
 def read_crop_scape_csv(path):
-    df = pandas.read_csv(path, engine='python', sep=", ")
+    df = pd.read_csv(path, engine='python', sep=", ")
```

Comment:
Might be better to remove the space from the delimiter for flexibility:
`df = pd.read_csv(path, engine='python', sep=",")`

Reply:
The CSV returned by cropspace uses ", " as the delimiter for all rows. If we made this change we would have to strip whitespace on all column names and values.